### ISSUE TITLE
[no-magic-numbers] False positive for negative readonly class properties

### ISSUE THREAD

**Issue by mustafa-cosar** [typescript-eslint/typescript-eslint#1070]
**Repro**
```JSON
{
  "rules": {
    '@typescript-eslint/no-magic-numbers': [
            'error',
            {
                'ignore': [-1, 0, 1, 2],
                'ignoreEnums': true,
                'ignoreNumericLiteralTypes': true,
                'ignoreReadonlyClassProperties': true
            }
        ],
    'no-magic-numbers': 'off',
  }
}
```

```TS
export class Class
{
    public static readonly NEGATIVE_NUMBER = -127;
}
```

**Expected Result**
No `@typescript-eslint/no-magic-numbers` error for either number

**Actual Result**
`[@typescript-eslint/no-magic-numbers] No magic number: -127.` but not for the positive number

**Additional Info**

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `2.3.2` |
| `@typescript-eslint/parser`        | `2.3.2` |
| `TypeScript`                       | `3.6.3` |
| `ESLint`                           | `6.5.1` |
| `node`                             | `v10.16.3` |
| `npm`                              | `6.12.0` |

**Comment by a-tarasyuk** [comment]
> but not for the positive number

I think, adding `+` will cause error too. 

```TS
public static readonly NEGATIVE_NUMBER = +127;
```

This is because `+127` is not `Literal`, it is `UnaryExpression`.

cc @bradzacher
